daprdocs/content/en/reference/cli/dapr-run.md

```diff
@@ -28,7 +28,7 @@ dapr run [flags] [command]
 | `--app-port`, `-p`             | `APP_PORT`           |                                                                                    | The port your application is listening on                                                            |
 | `--app-protocol`, `-P`         |                      | `http`                                                                             | The protocol Dapr uses to talk to the application. Valid values are: `http` or `grpc` |
 | `--app-ssl`                    |                      | `false`                                                                            | Enable https when Dapr invokes the application                                                       |
-| `--components-path`, `-d`      |                      | Linux/Mac: `$HOME/.dapr/components` <br/>Windows: `%USERPROFILE%\.dapr\components`   | The path for components directory                                                                  |
+| `--resources-path`, `-d`      |                      | Linux/Mac: `$HOME/.dapr/components` <br/>Windows: `%USERPROFILE%\.dapr\components`   | The path for components directory                                                                  |
```

Comment:
@pravinpushkar - Can you clarify here that components-path is not removed in this release, it is just marked as deprecated. In otherwords should we keep both here for now with a note to say components-path is deprecated?